### PR TITLE
[DYN-2476] First graph run does not show execution time in TuneUp +

### PR DESCRIPTION
**Pull request by ivaylo-matov**
Small PR that aims to address https://jira.autodesk.com/browse/DYN-2476 , as well as (partially) https://jira.autodesk.com/browse/DYN-4340 🤞

DYN-2476_1

DYN-0000_1

- TuneUp tracks changes in workspaces but only subscribes to these changes when it is enabled.
- When enabled, TuneUp forces the workspace to use the Manual run type to improve performance.
- Removed unnecessary updates to the UI to enhance responsiveness.
- Node execution time is now measured using Stopwatch for better accuracy.
- DataGrid headers retain their sorting order when the graph is re-executed. The sorting order reverts to the default when TuneUp is disabled, preparing for further improvements in https://jira.autodesk.com/browse/DYN-2475

### CODE REVIEW

**Comment by QilongTang** [comment]
Thanks @ivaylo-matov Would you resolve the merge conficts?

**Comment by ivaylo-matov** [comment]
All done @QilongTang 👍
There'll probably be more conflicts on the other PRs. I'll check them out later

File: TuneUp/ProfiledNodeViewModel.cs [hunk 1]
```diff
@@ -130,6 +132,11 @@ public string StateDescription
             }
         }
 
+        /// <summary>
+        /// The Stopwatch to measure execution time of this node
+        /// </summary>
+        public Stopwatch Stopwatch { get; set; }
```
Comment (reddyashish) [inline TuneUp/ProfiledNodeViewModel.cs:138]: Do we need this as public? Can we make this internal?

Comment (ivaylo-matov) [reply TuneUp/ProfiledNodeViewModel.cs]: Good point. It's internal now.

**Review by reddyashish — APPROVED** [review]

**Pull request merged**